pkg/gadgets/socket-collector/tracer/bpf/tcp4-collector.c

```diff
@@ -45,7 +45,20 @@ static int dump_tw_sock(struct seq_file *seq, struct tcp_timewait_sock *ttw)
 
 	socket_bpf_seq_print(seq, proto, tw->tw_rcv_saddr,
 		tw->tw_sport, tw->tw_daddr,
-		tw->tw_dport, tw->tw_substate, 42);
+		/*
+		 * tcp_timewait_sock represents socket in TIME_WAIT state.
+		 * Socket is this particular state are not associated with a
+		 * struct sock:
+		 * https://elixir.bootlin.com/linux/v5.15.12/source/include/linux/tcp.h#L442
+		 * https://elixir.bootlin.com/linux/v5.15.12/source/include/net/inet_timewait_sock.h#L33
+		 * Hence, they do not have an underlying file and, as a
+		 * consequence, no inode.
+		 *
+		 * Like /proc/net/tcp, we print 0 as inode number for TIME_WAIT
+		 * (state 6) socket:
+		 * https://elixir.bootlin.com/linux/v5.15.12/source/include/net/tcp_states.h#L18
+		 */
+		tw->tw_dport, tw->tw_substate, 0);
```

Comment:
What do you mean by this?

Reply:
I mean that commit [65eb48e](https://github.com/kinvolk/inspektor-gadget/pull/449/commits/65eb48eb13389b6391452c0eb4cc5e584a71bed8) is setting this to `42` but in the next commit [0c2e372](https://github.com/kinvolk/inspektor-gadget/pull/449/commits/0c2e3725f0a083f8978d8a62161a12120cc7e06c), you are changing that value to `0`. 

Could you please combine all the modifications related to this part in one single commit? It would make things easier when, in the future, one wants to understand when a given line was introduced.

Reply:
Oops! I will fix it!